### PR TITLE
Misc. Umbrella fixes

### LINKED ISSUE #62195: Umbrella is 11 cm length and in my rain coat...
### Describe the bug

I found an umbrella and was shocked by its basic melee parameters, but when I was needed to unload the coat to put it off and was asking would to store the umbrella in INVENTORY (to stop it wielding) I was shocked even more (actually, only after realizing that it is in the coat when started looking for where the umbrella is).

### Steps to reproduce

Look at it.

### Expected behavior

L = 1.5 seems ok, but
Weight = 0.14 kg and Length = 11 cm seem unnatural (btw 4 inches - it's transitive).

So it needs to be naturated.

### Screenshots

cdda1
cdda2

### Versions and configuration

Game Version: 0.F [64-bit]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by bean-b**
#### Summary
Bugfixes "Misc. Umbrella fixes"

#### Purpose of change
- closes https://github.com/CleverRaven/Cataclysm-DDA/issues/62195

#### Describe the solution
I changed the length of the umbrella to 101cm which i got from this diagram
image
I changed the weight to 411 grams which i got from this amazon listing
image

While i was at it, I noticed that the telescoping umbrella doesnt have any option to extend, and thus is always stuck in its retracted position. I changed it to act like the batons, with an extended and retracted modes.

I was also a bit confused as the option to wear it as armor, but i left it was a feature. I changed the protection to just be 1mm of plastic, which could represent like the plastic wrist strap, sense 0.5mm of alumnuim did'nt make much sense.

I also changed the to hit and damage of the retraced umbrella, as +1 to hit on a retracted umbrella felt wrong

#### Describe alternatives you've considered

Not doing this?

#### Testing

Spanwed in game, observed values, activited telescoping umbrella

### CODE REVIEW

File: data/json/items/tool/misc.json [hunk 1]
```diff
@@ -707,14 +707,14 @@
     "id": "teleumbrella",
     "type": "ARMOR",
     "category": "tools",
-    "name": { "str": "telescoping umbrella" },
-    "description": "A telescoping umbrella which collapses down for easy storage, useful for keeping dry when wielded.",
-    "weight": "118 g",
-    "volume": "500 ml",
+    "name": { "str": "telescoping umbrella (retracted)", "str_pl": "telescoping umbrellas (retracted)" },
```
Comment (JazzGlobal) [inline data/json/items/tool/misc.json:710]: I find it a bit odd that we're using both retracted and collapsed. Would make more sense to just use collapsed throughout

Comment (bean-b) [reply data/json/items/tool/misc.json:710]: yea, that makes sense, ill change it

**Comment by JazzGlobal** [comment]
I'm happy with this, only thing I can think of is that I don't have a way of validating this change does what it says it does without launching the game and finding an umbrella. Unless there are dev tools or a spawn menu I'm just unware of.

**Comment by bean-b** [comment]
> I'm happy with this, only thing I can think of is that I don't have a way of validating this change does what it says it does without launching the game and finding an umbrella. Unless there are dev tools or a spawn menu I'm just unware of.

>
cant you spawn an umbrella in the debug menu?

**Comment by JazzGlobal** [comment]
> > I'm happy with this, only thing I can think of is that I don't have a way of validating this change does what it says it does without launching the game and finding an umbrella. Unless there are dev tools or a spawn menu I'm just unware of.
> 
> > 
> 
> cant you spawn an umbrella in the debug menu?

Sure can, sorry! Looks good to me.

**Review by JazzGlobal — APPROVED** [review]

File: data/json/items/tool/misc.json [hunk 2]
```diff
@@ -707,14 +707,14 @@
     "id": "teleumbrella",
     "type": "ARMOR",
     "category": "tools",
-    "name": { "str": "telescoping umbrella" },
-    "description": "A telescoping umbrella which collapses down for easy storage, useful for keeping dry when wielded.",
-    "weight": "118 g",
-    "volume": "500 ml",
+    "name": { "str": "telescoping umbrella (collapsed)", "str_pl": "telescoping umbrellas (collapsed)" },
+    "description": "A telescoping umbrella which collapses down for easy storage, useful for keeping dry when wielded.  It is currently collapsed.",
```
Comment (cake-pie) [inline data/json/items/tool/misc.json:711]:
```suggestion
    "description": "A telescoping umbrella which has been collapsed down for easy storage.  May be used to keep dry when extended and wielded.",
```

File: data/json/items/tool/misc.json [hunk 3]
```diff
@@ -728,9 +728,35 @@
...
-    ]
+    ],
+    "use_action": { "menu_text": "Expand", "type": "transform", "target": "teleumbrella-extended", "msg": "You open your umbrella." }
+  },
+  {
+    "id": "teleumbrella-extended",
+    "type": "GENERIC",
+    "category": "tools",
+    "name": { "str": "telescoping umbrella (extended)", "str_pl": "telescoping umbrellas (extended)" },
+    "description": "A telescoping umbrella which collapses down for easy storage, useful for keeping dry when wielded.  It is currently extended.",
```
Comment (cake-pie) [inline data/json/items/tool/misc.json:742]:
```suggestion
    "description": "A telescoping umbrella currently in its extended form, useful for keeping dry when wielded.  May be collapsed down for easy storage.",
```

**Review by cake-pie — CHANGES_REQUESTED** [review]
With the telescoping umbrella changed to have explicit extended/retracted forms, I would expect  it can only be used to shield from the rain when actually extended and not when collapsed for storage.  Suggest removing `RAIN_PROTECT` from the collapsed version and to refine the descriptions to reflect this.

File: data/json/items/tool/misc.json [hunk 4]
```diff
@@ -707,30 +707,56 @@
...
         "encumbrance": 25,
         "coverage": 50,
         "covers": [ "hand_l", "hand_r" ],
-        "specifically_covers": [ "hand_wrist_l", "hand_wrist_r" ]
+        "specifically_covers": [ "hand_wrist_l", "hand_wrist_r" ],
+        "material": [ { "type": "plastic", "covered_by_mat": 100, "thickness": 1 } ]
       }
-    ]
+    ],
+    "use_action": { "menu_text": "Expand", "type": "transform", "target": "teleumbrella-extended", "msg": "You open your umbrella." }
```
Comment (cake-pie) [inline data/json/items/tool/misc.json:735]: For consistency with name and description of extended form.
```suggestion
    "use_action": { "menu_text": "Extend", "type": "transform", "target": "teleumbrella-extended", "msg": "You open your umbrella." }
```

**Pull request merged**